### ISSUE TITLE
MouseHunt AutoBot ENHANCED + REVAMP (version 2.3.6b) + MouseHunt AutoBot Additional thing

### ISSUE THREAD

**Issue by Marcus-Curtius** [nobodyrandom/mhAutobot#36]
AutoSolve not refreshing screen after solving the captcha. Using Chrome and https

**Comment by nobodyrandom** [comment]
Want to know if you are on fresh coat UI?

**Comment by Marcus-Curtius** [comment]
Yes, but changed back to classic, I’ll let you know if that resolves. Thank for the quick response. 


Nescit vox missa reverti

**Comment by nobodyrandom** [comment]
BTW, if you want, I have released 2.3.7b for test fix on beta UI. See if it fixes your issues :)

**Comment by nobodyrandom** [comment]
Main issue is that mh servers load slower than my script, and my script ends up thinking it is stuck after solving KR. So depending on how fast your internet is, you might get stuck on KR solved.

**Comment by Marcus-Curtius** [comment]
I do have a decent internet connection. I’ll try the new script today. Switching back to classic fixed the delay. Thanks again for the quick response. 


Nescit vox missa reverti
